src/telemetry/telemetry.c

```diff
@@ -329,6 +335,7 @@ telemetry_main()
 
 	http_response_state_destroy(rsp);
 
+cleanup:
 	if (started)
 		CommitTransactionCommand();
```

Comment:
Should this txn commit in the success case and abort in the failure case?

Reply:
done